templates/hello-world/package.json

```diff
@@ -5,11 +5,10 @@
   "version": "0.0.0",
   "scripts": {
     "build": "shopify hydrogen build",
-    "dev": "shopify hydrogen dev --codegen",
+    "dev": "shopify hydrogen dev",
```

Comment:
Yeah this caught my attention too.

Reply:
This is in `hello-world` template. This template is not yet even updated to Vite, and it does not have generated types for queries. Instead of adding the codegen dependencies in this template, I'm just removing the `--codegen` flag to keep it minimal 🤔